nix-libs/nodeLib/tools/execute-install-scripts

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
Can we generate this script from the Nix code so that we can interpolate the shebang line?

Reply:
The `node-build-tools` definition calls `patchShebangs`, so when built all of the shebangs contain full nix store paths.

Reply:
Yeah I think that's fine.